### PR TITLE
Feature/guidelines

### PR DESCRIPTION
**Pull request by fxstr**
Basic functionality for guidelines; some error handling can (and will) still be improved.

### CODE REVIEW

File: www/src/js/components/drawer/DrawerGuidelineContent.jsx [hunk 1]
```diff
@@ -9,15 +9,8 @@ export default @observer class DrawerGuidelineContent extends React.Component {
...
-    getAntibioticTherapyMarkup(antibioticTherapy) {
-        return { __html: marked(antibioticTherapy.markdownText) };
-    }
-
-    /**
-     * See this.getAntibioticTherapyMarkup()
-     */
-    getDiagnosisMarkup(diagnosis) {
-        return { __html: marked(diagnosis.markdownText) };
+    generateMarkdownFromHtml(content) {
```
Comment (tobiaskneubuehler) [inline www/src/js/components/drawer/DrawerGuidelineContent.jsx]: Isn't it «generateHtmlFromMarkdown» :)?

Comment (fxstr) [reply www/src/js/components/drawer/DrawerGuidelineContent.jsx]: 🙈

File: www/src/config/config.live.js [hunk 2]
```diff
@@ -9,6 +9,14 @@ export default Object.freeze({
         countries: 'generics.country',
         ageGroups: 'generics.ageGroup',
         hospitalStatus: 'generics.hospitalStatus',
+        guidelineBaseUrl: 'http://guidelines.infect.info/v1/',
```
[the remark this answers is not in the document]
Comment (fxstr) [reply www/src/config/config.live.js]: Yes, we should do that. Configs will change with the new release (GCP), let's do it then.

File: www/src/js/main.js [hunk 3]
```diff
@@ -158,7 +164,7 @@ function renderReact() {
     );
 
     ReactDOM.render(
-        <Notifications />,
+        <Notifications errors={app.errorHandler.errors} />,
```
Comment (tobiaskneubuehler) [inline www/src/js/main.js]: Did you test this? I had some problems in the past if I passed the observable directly

Comment (fxstr) [reply www/src/js/main.js]: It would be if it was a primitive – right?

**Pull request merged**